shared/src/msi-installer/Tracer/Files.wxs

```diff
@@ -4,35 +4,35 @@
      xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
   <?include $(sys.CURRENTDIR)\Config.wxi?>
   <Fragment>
-    <ComponentGroup Id="Tracer.Files" Directory="Tracer">
+    <ComponentGroup Id="Tracer.Files" Directory="INSTALLFOLDER">
     </ComponentGroup>
 
     <?if $(var.Win64) = yes ?>
-    <ComponentGroup Id="Tracer.Files.Native.64" Directory="Tracer.win_x64">
+    <ComponentGroup Id="Tracer.Files.Native.64" Directory="INSTALLFOLDER.win_x64">
       <Component Win64="yes">
         <File Id="Datadog.Trace.ClrProfiler.Native"
```

Comment:
Not required, but might be useful:

```suggestion
        <File Id="Datadog.Tracer.Native"
```

Reply:
I wasn't sure if it was safe to change these IDs which is why I left them. Good to know it is, thanks! 🙂